### PR TITLE
feat: add button download full history

### PR DESCRIPTION
**Pull request by GusBaamonde**
kreitech boys look @josedutra @bravoy93:
https://cdn.fromdoppler.com/doppler-ui-library/build1127/subscriber-history.html

descarga-historial

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/templates/subscriber-history.html [hunk 1]
```diff
@@ -2795,6 +2795,13 @@ <h3>Estado GDPR del suscriptor</h3>
               </table>
             </div>
           </div>
+
+          <div class="dp-cta-links">
+            <a href="#"
+              ><span class="ms-icon icon-download"></span> DESCARGAR HISTORIAL
+              COMPLETO</a
```
Comment (andresmoschini) [inline assets/templates/subscriber-history.html:2802]:
```suggestion
              ><span class="ms-icon icon-download"></span> Descargar historial
              completo</a
```

**Comment by andresmoschini** [comment]
👍

File: assets/templates/subscriber-history.html (hunk 1 above)
Comment (GusBaamonde) [reply assets/templates/subscriber-history.html:2802]: the link has the uppercase, so it's the same

Comment (andresmoschini) [reply assets/templates/subscriber-history.html:2802]: Right, using uppercase is a style decision, so, the content should not be affected.

**Comment by cbernat** [comment]
👍

**Comment by josedutra** [comment]
Looks good! 👍

**Comment by bravoy93** [comment]
thanks!!

**Pull request merged**